.github/patch-initial-checklist.md

```diff
@@ -90,3 +81,17 @@ This only needs done if the patch release needs to be included in WooCommerce Co
   - Verify and make any additional edits to the pull request description for things like: Changelog to be included with WooCommerce core, additional communication that might be needed elsewhere, additional marketing communication notes that may be needed etc.
   - After the checklist is complete and the testing is done, it will be up to the WooCommerce core team to approve and merge the pull request.
 * [ ] Make sure you join the `#woo-core-releases` Slack channel to represent Woo Blocks for the release of WooCommerce core this version is included in.
+
+## Publish Posts
+
+You only need to post public release announcements and update relevant public facing docs if this patch release is deployed to WP.org. Otherwise, you can skip this section.
+
+* [ ] Post release announcement on [WooCommerce Developer Blog](https://developer.woocommerce.com/category/release-post/woocommerce-blocks-release-notes/).
+  - Ping porters from each team to know which changelog entries need to be highlighted. Ask them to write a short text and optionally provide a screenshot. They can use previous posts for inspiration, we usually try to highlight new features or API changes.
+  - Ensure the release notes are included in the post verbatim.
+  - Don't forget to use category `WooCommerce Blocks Release Notes` for the post.
```

Comment:
Should we do a final proof read with both teams before posting? From what I've noticed there are usually small issues that are fixed in this step, like versions, links etc

Reply:
I was thinking having the porters write the highlights and the release lead merging them in the post would be enough. But I don't have a strong opinion.